config/source/300-rabbitmqsource.yaml

```diff
@@ -126,18 +117,9 @@ spec:
               queueConfig:
                 description: QueueConfig config for rabbitmq queues
                 properties:
-                  autoDelete:
-                    type: boolean
-                  durable:
-                    description: Queue is Durable or not
-                    type: boolean
                   name:
                     description: Name of the queue to bind to; required value.
                     type: string
-                  routingKey:
-                    description: Routing key of the messages to be received. Multiple
-                      routing keys can be specified separated by commas. e.g. key1,key2
-                    type: string
                 required:
                 - name
                 type: object
```

Comment:
Plan was to move retry and other deliverySpec options into a `delivery` struct similar to the Broker.

Reply:
Yep that sounds good, but let's do it on another PR if that is ok